components/x-timeline-feed/readme.md

```diff
@@ -0,0 +1,55 @@
+# x-timeline-feed
+
+This component renders a list of articles in reverse chronological order, grouped by day, according to the user's timezone.
+It will optionally group today's articles into "latest" and "earlier" too.
+
+## Installation
+
+This module is compatible with Node 6+ and is distributed on npm.
+
+```bash
+npm install --save @financial-times/x-timeline-feed
+```
+
+The [`x-engine`][engine] module is used to inject your chosen runtime into the component. Please read the `x-engine` documentation first if you are consuming `x-` components for the first time in your application.
+
+[engine]: https://github.com/Financial-Times/x-dash/tree/master/packages/x-engine
+
+
+## Usage
+
+The components provided by this module are all functions that expect a map of [properties](#properties). They can be used with vanilla JavaScript or JSX (If you are not familiar check out [WTF is JSX][jsx-wtf] first). For example if you were writing your application using React you could use the component like this:
+
+```jsx
+import React from 'react';
+import { TimelineFeed } from '@financial-times/x-timeline-feed';
+
+// A == B == C
+const a = TimelineFeed(props);
+const b = <TimelineFeed {...props} />;
+const c = React.createElement(TimelineFeed, props);
+```
+
+All `x-` components are designed to be compatible with a variety of runtimes, not just React. Check out the [`x-engine`][engine] documentation for a list of recommended libraries and frameworks.
+
+[jsx-wtf]: https://jasonformat.com/wtf-is-jsx/
+
+### Properties
+
+Feature              | Type   | Notes
+---------------------|--------|----------------------------
+`articles`           | Array  | (Mandatory) Array of objects, in Teaser format, representating articles to render. The articles should be in newest-first order.
+`timezoneOffset`     | Number | (Default = 0) Minutes to offset article publish times in order to display in user's timezone
```

Comment:
As just discussed, perhaps this could default to the runtime clock as `localTodayDate` does. Makes it simpler for client-side usage.